code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -2944,7 +2944,7 @@
 
 /obj/item/weapon/reagent_containers/food/snacks/sliceable/bread
 	name = "Bread"
-	icon_state = "Some plain old Earthen bread."
```

Comment:
🤔

Reply:
Turns out bread had a description all this time, but it was using the icon state variable, which was overwritten the very next line, so it was never picked up on.